website/source/docs/consul-enterprise/index.html.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: "docs"
+page_title: "Consul Enterprise"
+sidebar_current: "docs-consul-enterprise"
+description: |-
+  Consul Enterprise features a number of capabilities beyond the open
+  source offering that may be beneficial in certain workflows.
+---
+
+# Consul Enterprise
+
+Consul Enterprise simplifies operations by automating workflows. It adds support for microservices deployments across complex network topologies. It also increases both scalability and resilience. Features include:
+
+- [Automated Backups](/docs/consul-enterprise/backups)
```

Comment:
These should end with /index.html (it'll work without, but we get penalized for duplicate content if we don't link directly to the page)